src/Umbraco.Cms.Integrations.Search.Algolia/Umbraco.Cms.Integrations.Search.Algolia.csproj

```diff
@@ -24,7 +24,7 @@
 	<ItemGroup>
 		<PackageReference Include="Algolia.Search" Version="6.13.0" />
 		<PackageReference Include="Umbraco.Cms.Web.Website" version="[10.3.0,)" />
-		<PackageReference Include="Umbraco.Cms.Web.BackOffice" version="[10.3.0,)" />
+		<PackageReference Include="Umbraco.Cms.Web.BackOffice" version="[10.6.0,)" />
```

Comment:
If updating the reference - which is fine - you should update the other `Umbraco.Cms` ones to to be on a consistent version.